### PR TITLE
Testing updates

### PR DESCRIPTION
**Pull request by lesv**
@tswast First of a couple changes.

### CODE REVIEW

**Comment by lesv** [comment]
Never mind, just saw something stupid.

**Comment by lesv** [comment]
@tswast PTAL

**Comment by tswast** [comment]
Thanks. LGTM.

File: travis.sh [hunk 1]
```diff
@@ -19,6 +19,9 @@ set -x
 set -o pipefail
 shopt -s globstar
 
+add_ppa 'ppa:cwchien/gradle'
+sudo apt-get -qqy install gradle-3.0
```
Comment (tswast) [inline travis.sh]: Travis might not like this. Probably have to add this stuff to the `.travis.yml` instead https://docs.travis-ci.com/user/installing-dependencies/#Installing-Packages-with-the-APT-Addon (or change to VM-based builder with `sudo: required` in `.travis.yml` https://docs.travis-ci.com/user/ci-environment/#Virtualization-environments.

**Pull request merged**